cmd/dataset/main.go

```diff
@@ -107,6 +114,41 @@ func handle() {
 		}
 	}))
 
+	if pprofAddr != "" {
```

Comment:
How about to create a common function to launch pprof?  This piece of code has been repeated several times. Just a suggestion, WDYT?

Reply:
yes, I will create a common function now